yaml-tests/src/test/resources/aggregate-index-tests-count-empty.metrics.yaml

```diff
@@ -3,9 +3,9 @@ agg-index-tests-count-empty:
     explain: 'AISCAN(MV1 <,> BY_GROUP -> [_0: VALUE:[0]]) | MAP (_ AS _0) | ON EMPTY
         NULL | MAP (coalesce_long(_._0._0, promote(0l AS LONG)) AS _0)'
     task_count: 434
-    task_total_time_ms: 13
+    task_total_time_ms: 19
```

Comment:
Does this PR need to update all of the checked in metrics? It we can avoid doing it, it would be nice just to avoid churn on these files if it's not needed

Reply:
Yes -- there was a mistake int the previous PR that it also kept metrics for `explainContains:` which causes problems for downstream tests.

Reply:
So I am recreating all metrics here since I don't want any of the `explainContains:` metrics to be kept at all.